app/main.py

```diff
@@ -7,6 +7,11 @@
 from services.logger import setup_logger
 from api.error_utilities import ErrorResponse
 
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
```

Comment:
In the off-chance that we need to refresh the dotenv variables, we should clear the cache using `load_dotenv(find_dotenv())`